include/drjit-core/optix.h

```diff
@@ -30,14 +31,40 @@ extern JIT_EXPORT void *jit_optix_lookup(const char *name);
  */
 #define jit_optix_check(err) jit_optix_check_impl((err), __FILE__, __LINE__)
 extern JIT_EXPORT void jit_optix_check_impl(int errval, const char *file,
-                                              const int line);
+                                            const int line);
 
-/// Inform Dr.Jit about a partially created OptiX pipeline
-extern JIT_EXPORT void
-jit_optix_configure(const OptixPipelineCompileOptions *pco,
-                    const OptixShaderBindingTable *sbt,
-                    const OptixProgramGroup *pg,
-                    uint32_t pg_count);
+/**
+ * \brief Inform Dr.Jit about a partially created OptiX pipeline
+ *
+ * This function creates a JIT variable responsible for the lifetime management
```

Comment:
Would you mind adding a small example of how `jit_optix_configure_pipeline` and `jit_optix_configure_sbt` relate to each other (otherwise users will have to dig into the Mitsuba code to understand how they are used).